augur/filter.py

```diff
@@ -721,6 +734,7 @@ def apply_filters(metadata, exclude_by, include_by):
     For example, filter data by minimum date, but force the include of strains
```

Comment:
This "For example" line gets pulled into the "Returns" block of the rendered docs. I think another blank line before this line would fix the issue.

Reply:
Ah, good eye. Fixed in the force push.